src/containers/pages/IRS/assignments/index.tsx

```diff
@@ -40,6 +40,9 @@ reducerRegistry.register(IRSPlansReducerName, IRSPlansReducer);
 
 const OpenSrpPlanService = new OpenSRPService('plans');
 
+/** Plans filter selector */
+const plansArraySelector = makePlansArraySelector('planRecordsById');
```

Comment:
Should `planRecordsById` be moved to `constants`? Or maybe even the `plans` dux module?

Reply:
@cKellyDesign Have moved `planRecordsById` to constants.